### ISSUE TITLE
ISSUE WITH WIFITE ON ALFA AWS036ACH AC 1200

### ISSUE THREAD

**Issue by randomuserr12** [derv82/wifite2#473]
Hello, i've purchased the " ALFA AWUS036ACH AC1200 ADAPTATEUR USB DUAL BAND" on Amazon and i can't get it to work properly if anyone can help me  especially in wifite, aircrack

Look i've tried to install last driver, update package everything but this is what happened when i run "sudo wifite"
 
![image](https://github.com/derv82/wifite2/assets/108004589/c1e57b8e-dd97-4487-aaef-3699888a11f9)
                                                                              

After that i tried to run "sudo airmon-ng start wlan0", and this is answer i got  : 
![image](https://github.com/derv82/wifite2/assets/108004589/12524366-a3bc-4e99-8327-69ddc034efb8)

Also there's ifconfig , iwconfig before you asked me i really dont know what to do at the moment : 

![image](https://github.com/derv82/wifite2/assets/108004589/d73c5f61-b18e-459f-bc07-c9e92fdc1548)

(Also same issue on aircrack to scan network)

![image](https://github.com/derv82/wifite2/assets/108004589/6f956d75-2443-4c01-8f4d-9b9b23a574e6)

Tell me, if you need more information im on Kali every drivers are updated, and driver is perfectly installed

Ty.

**Comment by seferino-p8** [comment]
@Alby2000 does your Wi-Fi adapter support monitor mode ?

**Comment by randomuserr12** [comment]
> @Alby2000 does your Wi-Fi adapter support monitor mode ?

Yes it does but anyways I'll buy a new one do you recommended any specific Wi-Fi adapter ? Ty

**Comment by seferino-p8** [comment]
@Alby2000 yes ALFA AWUS036NHA
